components/style/properties/shorthand/position.mako.rs

```diff
@@ -138,6 +138,107 @@
 
 </%helpers:shorthand>
 
+% for kind in ["row", "column"]:
+<%helpers:shorthand name="grid-${kind}" sub_properties="grid-${kind}-start grid-${kind}-end"
+                    spec="https://drafts.csswg.org/css-grid/#propdef-grid-${kind}"
+                    products="gecko">
+    use values::specified::GridLine;
+    use parser::Parse;
+
+    pub fn parse_value(context: &ParserContext, input: &mut Parser) -> Result<Longhands, ()> {
```

Comment:
Presumably we could implement one in terms of the other, which would be less code, could you leave a comment there wrt that?

(for reference, I mean implement grid-row and doing something like the following for grid-column: `grid_row::parse_value(context, input).map(|l| Longhands {..})`)